CONTRIBUTING.rst

```diff
@@ -84,7 +84,7 @@ The main one is the SunPy repository with where all the known `issues`_ with Sun
 Each issue should have a series of labels that provide information about the nature of the issue.
 If you find an issue you'd like to work on, please make sure to add a comment to let people know that you are working on it! This will make it less likely that effort is duplicated.
 
-If you are unsure where to start we suggest the `Package Novice label`_.
+If you are unsure where to start we suggest the `Good First Issue`_.
```

Comment:
```suggestion
If you are unsure where to start we suggest the `Good First Issue`_ label.
```

You also need to change the URL and name of this at line 113.

Reply:
https://github.com/sunpy/sunpy/issues?utf8=✓&q=is:issue+is:open+label:"Good+First+Issue" will this work